.agent-versions.json

```diff
@@ -2,8 +2,7 @@
   "testVersions": [
     "8.15.2-SNAPSHOT",
     "8.15.1",
-    "8.14.3",
-    "7.17.24",
-    "7.17.24-SNAPSHOT"
+    "8.15.0",
+    "7.17.25-SNAPSHOT"
```

Comment:
I don't think this change makes sense now. The automation is using 9.0 as a target version for getting the versions and all the rules are shifted now. So, the previous major is now 8.x instead of 7.x and 7.17.14 got completely removed. The snapshots remained because of another rule that requires N last snapshots for branches.

Reply:
So what should we do with these PRs? Should we disable the automation until 9.0.0-SNAPSHOT is available? Should we just close this PR and wait for the next run?

Reply:
Close and wait for another one, I thought I closed this one, my bad.